.env.example

```diff
@@ -13,20 +14,27 @@ DB_USERNAME=homestead
 DB_PASSWORD=secret
 
 CACHE_DRIVER=redis
+FILESYSTEM_DRIVER=public
+QUEUE_CONNECTION=sync
 SESSION_DRIVER=redis
-STORAGE_DRIVER=public
-QUEUE_DRIVER=sync
 
-REDIS_HOST=127.0.0.1
-REDIS_PASSWORD=null
-REDIS_PORT=6379
+REDIS_URL=redis://h:null@127.0.0.1:6379
+
+# AWS
+AWS_ACCESS_KEY=
+AWS_REGION=us-east-1
+AWS_SECRET_KEY=
+AWS_S3_BUCKET=
+AWS_S3_URL=
```

Comment:
This is another one we'll need to update in our Terraform config ([here](https://github.com/DoSomething/infrastructure/blob/246a5c8333226d50419a9affc50617084448431f/components/s3_bucket/main.tf#L204-L205) and [here](https://github.com/DoSomething/infrastructure/blob/246a5c8333226d50419a9affc50617084448431f/components/iam_app_user/main.tf#L31-L32)). Like the others, we'll want to keep that old environment variables until we've confirmed everything is updated to use these new names.

Reply:
Since we're changing these now, I'd like to use `AWS_ACCESS_KEY_ID` in place of `AWS_ACCESS_KEY` & `AWS_SECRET_ACCESS_KEY` in place of `AWS_SECRET_KEY` (for consistency with [Laravel's defaults](https://github.com/laravel/laravel/blob/a7b70fcc310fee9d78448e078fef9003150ee8aa/.env.example) and [Amazon's CLI environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).

Reply:
I think we'll also want to maintain separate `AWS_REGION` variables for different services (e.g. `AWS_S3_REGION` and `AWS_SQS_REGION`) since those might be in different regions. (I mean, 100% unlikely, but it's possible!)

Reply:
> Since we're changing these now, I'd like to use AWS_ACCESS_KEY_ID in place of AWS_ACCESS_KEY & AWS_SECRET_ACCESS_KEY in place of AWS_SECRET_KEY (for consistency with Laravel's defaults and Amazon's CLI environment variables.

Sounds good! I was debating changing those or not since I was already changing a bunch and wasn't sure how much of a pain it'd be to change. But you're right, since we're doing the rest we might as well and definitely prefer to be consistent with Laravel's defaults 👍

Reply:
@DFurnes changing these up and doing your recommendation of `AWS_S3_REGION` and `AWS_SQS_REGION` but wondering what's the call for the [aws config file](https://github.com/DoSomething/rogue/blob/8bcfe954aa20233afe0377cedd45d5a03bb218de/config/aws.php#L20). It just generally specifies `region`, I'm assuming it's the S3 one, but want to confirm 🤔

Reply:
Also I guess for [this one](https://github.com/DoSomething/rogue/blob/8bcfe954aa20233afe0377cedd45d5a03bb218de/config/services.php#L48), but we don't use SES so... 🤷 

Maybe 3 items? `AWS_REGION`, `AWS_S3_REGION`, & `AWS_SQS_REGION`?

Reply:
Hmm.. I'm not sure if we actually need that config file or not... might be interesting to try removing it & see if anything breaks. And yeah, I think those look good. Can't hurt to add an `AWS_SES_REGION` I guess and just not set it (maybe it defaults to `us-east-1`)?

Reply:
I double-checked and we don't change any of the default `aws` config options that are in the package so I do think you're right and this `config/aws.php` file is redundant. Removing and then adding those other `AWS_*` env vars.

Reply:
Ok, since there's also a setting for `dynamodb` cache setting I didn't want to add `AWS_DYNAMODB_REGION` as well, so I just added:

```
AWS_REGION=us-east-1
AWS_S3_REGION=us-east-1
AWS_SQS_REGION=us-east-1
```

Specific region env var for S3 and SQS since we use those and then everything else can just refer to the default `AWS_REGION`. If we need something more specific in the future we can always revise.